stdlib/public/Glibc/CMakeLists.txt

```diff
@@ -1,18 +1,25 @@
 set(sources
-  module.map
+  module.map.in
   )
 set(output_dir "${SWIFTLIB_DIR}/glibc")
 
-set(commands
-    COMMAND ${CMAKE_COMMAND} "-E" "make_directory" "${output_dir}")
-set(outputs)
+# Set correct paths to glibc headers
+set(GLIBC_INCLUDE_PATH "/usr/include")
+set(arch_dir "x86_64-linux-gnu")
```

Comment:
Also, as a side note: At some point it there will probably be a better way to get this path: https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=796545